.github/workflows/try-release-daily.yml

```diff
@@ -0,0 +1,69 @@
+name: Try building a release daily
+# Build a release tarball for all supported branches to verify the code is releasable.
+
+on:
+  schedule:
+    - cron: 0 3 * * *
+  # be able to start this action manually from a actions tab when needed
+  workflow_dispatch:
```

Comment:
I'm thinking if this should not be part of the PRs instead of daily. The main reason is that the daily runs are easy to miss and also that we could discover our mistake before merge.

Reply:
ljanda says: use languages in a bullet list here: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/8.6_release_notes/index#international_languages